site/src/examples/omezarr/render-utils.ts

```diff
@@ -48,8 +50,9 @@ export function buildConnectedRenderer(
     const client = cache.registerClient<Thing, Record<string, Tex>>({
         cacheKeys: (item) => {
             const channelKeys = Object.keys(item.settings.channels);
-            return channelKeys.reduce((chans, key) => {
-                return { ...chans, [key]: renderer.cacheKey(item.tile, key, item.dataset, item.settings) };
```

Comment:
are you making this changes for performance reasons?

Reply:
I personally think the linter is full of **it, but I suppose these changes are fine